docs/config.md

```diff
@@ -1397,6 +1398,43 @@ Default: `auto` (DHT + IPNI)
 
 Type: `optionalString` (`null`/missing means the default)
 
+
+### `Routing.AcceleratedDHTClient`
+
+Utilizes an alternative DHT client that searches for and maintains more information
+about the network in exchange for being more performant.
+
+This is not compatible with `Routing.Type` `custom`. If you are using composable routers
+you can configure this individualy on each router.
+
+When it is enabled:
+- DHT operations should complete much faster than with it disabled
+- A batching reprovider system will be enabled which takes advantage of some
+  properties of the experimental client to very efficiently put provider records into the network
+- The standard DHT client (and server if enabled) are run alongside the accelerated DHT client
+- The operations `ipfs stats dht` will default to showing information about the accelerated DHT client
```

Comment:
We say the word "default".  Is there a way for a user to change this when the accelerated DHT client is enabled?

Reply:
`ipfs stats dht` is default, `ipfs stats dht wansever` is when the user overrides.

Reply:
`ipfs stats dht` is indeeed a shorthand for `ipfs stats dht wan`. I agree with you the comment is weird.
I think the point is that by default `ipfs stats dht` dumps your `wan` routing table buckets which all have up to ~20 nodes.
While with `ipfs stats dht` it shows 1 single buckets that have ~20k nodes (or whatever big the network is).